config/config.go

```diff
@@ -144,6 +153,10 @@ func NewConfig() (c *Config) {
 	c.Products.Istio.IstioSidecarAnnotation = strings.TrimSpace(getDefaultString(EnvIstioSidecarAnnotation, "sidecar.istio.io/status"))
 	c.Products.Istio.UrlServiceVersion = strings.TrimSpace(getDefaultString(EnvIstioUrlServiceVersion, "http://istio-pilot:9093/version"))
 
+	// Token Configuration
+	c.Token.Secret = strings.TrimSpace(getDefaultString(EnvTokenSecret, "kiali"))
+	c.Token.ExpirationHours = getDefaultInt(EnvTokenExpirationHours, 2)
```

Comment:
Higher default for convenience, like 10 hours?

Reply:
ok